apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -431,6 +437,9 @@ export function buildMachine(
         on: {
           NO_PAPER_ANYWHERE: 'resetting_state_machine_after_success',
         },
+        after: {
+          [DELAY_BEFORE_DECLARING_REAR_JAM_MS]: 'jammed',
```

Comment:
It might be the way I'm forcing rear jams - it results in ~1 cm of the paper hanging in the rear output, with none of the paper inside the device. In other jam situations the paper is at least 2 inches inside the device. My guess is the jam sensors are internal only.

I haven't found a way to force a rear jam that results in an internal jam. The motors to eject to rear are pretty strong; even if I put my hand directly on the rear output to block it, the paper crunches up rather than get stuck.

Reply:
Thanks for the context, makes sense